modules/nodes-edge-remote-workers-strategies.adoc

```diff
@@ -99,18 +101,22 @@ metadata:
+      - "1m"
 ----
-<1> Specify the amount of time that the controller manager checks the state of each node that is associated with this machine config. The default is `5s`.
+<1> Specify the type of node to apply this `KubeletConfig` objects using the label from the `MachineConfig` object.
+<2> Specify the frequency that the kubelet checks the status of a node associated with this `MachineConfig` object. The default value is `10s`. If you change this default, the `node-status-report-frequency` value is changed to the same value.
+<3> Specify the frequency that kubelet reports the status of a node associated with this `MachineConfig` object. The default value is `1m`.
 
 This parameter works with the `node-monitor-grace-period` and the `pod-eviction-timeout` parameters, which are not configurable. 
 
-* The `node-monitor-grace-period` parameter specifies how long the {product-title} waits after a node associated with this machine config is marked `Unhealthy` if the controller manager cannot reach the node. Workloads on the node continue to run after this time. If the remote worker node rejoins the cluster after the `node-monitor-grace-period` expires, pods continue to run. New pods can be scheduled to that node. The `node-monitor-grace-period` interval is `40s`.
+* The `node-monitor-grace-period` parameter specifies how long the {product-title} waits after a node associated with this `MachineConfig` object is marked `Unhealthy` if the controller manager does not receive the node heartbeat. Workloads on the node continue to run after this time. If the remote worker node rejoins the cluster after the `node-monitor-grace-period` expires, pods continue to run. New pods can be scheduled to that node. The `node-monitor-grace-period` interval is `40s`. The `node-status-update-frequency` value must be lower than the `node-monitor-grace-period` value.
 
-* The `pod-eviction-timeout` parameter specifies the amount of time {product-title} waits after marking a node that is associated with this machine config as `Unreachable` to start marking pods for eviction. Evicted pods are rescheduled on other nodes. If the remote worker node rejoins the cluster after `pod-eviction-timeout` expires, the pods running on the remote worker node are terminated because the node controller has evicted the pods on-premise. Pods can then be rescheduled to that node. The `pod-eviction-timeout` period is `5m0s`.
+* The `pod-eviction-timeout` parameter specifies the amount of time {product-title} waits after marking a node that is associated with a `MachineConfig` object as `Unreachable` to start marking pods for eviction. Evicted pods are rescheduled on other nodes. If the remote worker node rejoins the cluster after `pod-eviction-timeout` expires, the pods running on the remote worker node are terminated because the node controller has evicted the pods on-premise. Pods can then be rescheduled to that node. The `pod-eviction-timeout` period is `5m0s`.
```

Comment:
Just want to make sure you intended to make the edit to `a MachineConfig object` instead of `this MachineConfig object` (as was done on line 117). It seems the param would only affect the MC object it was defined for.